2294/3/helpers/azure.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'scraper.rb'
+require_relative 'form_comments.rb'
+require 'net/https'
+require 'uri'
+require 'json'
+
+# Take messages from sorted by onliner's rating array
+class PackCreator
+  def initialize
+    @array_of_objects = Analyze.new.sort_array_of_objects
```

Comment:
don't u want to pass these objects as `new` argument? 
And give it more meaningful name